src/store/messages/saga.ts

```diff
@@ -477,9 +461,15 @@ export function* getPreview(message) {
   return yield call(getLinkPreviews, link[0].href);
 }
 
-function* syncChannelsTask(action) {
-  while (yield select(rawShouldSyncChannels(action.payload.channelId))) {
-    yield call(fetchNewMessages, action);
+function* pollForPublicChannelMessages() {
+  while (true) {
+    const currentUser = yield select(currentUserSelector());
+    const isPublicZOS = !currentUser?.id;
+    const activeChannelId = yield select(activeChannelIdSelector);
+
+    if (isPublicZOS && activeChannelId) {
+      yield call(fetchNewMessages, activeChannelId);
```

Comment:
same question here, curious about how the polling would work i am a logged in user, but in multiple networks (so i can see the normal view with all the public + private channels)

Reply:
It won't poll. You don't have to because real time events handle everything.